_directory/data/olien/data.yml

```diff
@@ -0,0 +1,29 @@
+---
+# name (required)
+# may be your real name or your nick
+name: Oliver Kreischer
+
+# geo location (required)
+# provide your favored level of detail by number of digits (like 50.107811 vs 50.11)
+# you may use services like openstreetmap.org to find out your geo location
+latitude: 51.399
+longitude: 7.190
+
+# bio
+# describe yourself, your involvement in the project and the community or whatever is relevant for you.
+# should be 300 chars at maximum. no markup or line breaks!
+bio: “Hi, I'm Oliver (olien) and I've been working with REDAXO since 2006. It's still fun every day! I also take care of the eggnog on the REDAXO days.”
```

Comment:
This means you no longer work with redaxo?

Reply:
Schreibs doch auf deutsch @olien oder ist englisch Bedingung @schuer ?

Reply:
@staabm For my opinion the wording is correct. Have been working since = he’s still working.
@tbaddade Kann auch deutsch sein, klar, just keep in mind the project wants to open for non-german speakers.